tests/test-env-check.sh

```diff
@@ -26,20 +26,11 @@ _END_
 
 if [ $? = 1 ]; then
     echo "Python 3.6 or newer is not installed/enabled."
-    return
+    exit 1
 else
     echo "Python 3.6 or newer detected!"
 fi
 
-# check for python development header -- for autest
-python3-config &> /dev/null
-if [ $? = 1 ]; then
-    echo "python3-dev/devel detected!"
-else
-    echo "python3-dev/devel is not installed. "
-    return
-fi
-
```

Comment:
Huh... on my fedora:38 system, `pipenv install` fails without python3-devel. I guess we should leave this check in.